### PR TITLE
Flu reference sequences

### PR DESCRIPTION
**Pull request by rneher**
this cleans a few thinks up and includes reference sequences prior to the cut off date. this will also fix the root, which otherwise is sometimes rather ambiguous

### CODE REVIEW

File: flu/flu.process.py [hunk 1]
```diff
@@ -31,17 +31,25 @@ def make_config (prepared_json, args):
...
         "clean": args.clean,
         "pivot_spacing": 1.0/12,
         "timetree_options": {
-            "Tc": 0.03
+            "Tc": 0.03,
+            # "confidence":True,
+            # "use_marginal":True
+        },
+        "newick_tree_options":{
+            "raxml":False
```
Comment (huddlej) [inline flu/flu.process.py]: Do we actually want RAxML turned off by default?

File: base/tree.py [hunk 2]
```diff
@@ -182,14 +184,16 @@ def timetree(self, Tc=0.01, infer_gtr=True, reroot='best', resolve_polytomies=Tr
             else:
                 node.attr = {'num_date':node.numdate}
             if confidence:
-                node.attr["num_date_confidence"] = sorted(to_numdate(node.marginal_inverse_cdf([0.05,0.95])))
+                node.attr["num_date_confidence"] = sorted(self.tt.get_max_posterior_region(node, fraction=0.9))
```
Comment (huddlej) [inline base/tree.py]: Is this replacement method call just a new name/API for the original call or is it a qualitatively different approach? I don't know enough about treetime yet to know myself. Maybe @trvrb knows?

File: flu/flu.process.py (hunk 1 above)
Comment (trvrb) [inline flu/flu.process.py]: I was making some revisions in parallel. I think that RAxML should be on by default and can be switched off when doing quicker testing for speed. I had set this to `True`.

File: base/tree.py (hunk 2 above)
Comment (trvrb) [inline base/tree.py]: Just by naming I think this is returning the exact same thing. The 90% max posterior will be equivalent to the 5% to 95% bounds on the inverse CDF.

**Comment by rneher** [comment]
the confidence intervals are mostly the same, but differ when they are peaked on the boundary (say a zero mutation branch). the max posterior interval contains the 90% of highest posterior probability, while the 5,95 the 90% "middle" of the distribution. sorry about forgetting the raxml switch (I have yet to be convinced though that raxml is actually better...)

**Pull request merged**